help/view-and-edit-your-message-drafts.md

```diff
@@ -1,37 +1,90 @@
-# View and edit your message drafts
+# Message drafts
 
-When you close the compose box, Zulip stores the content in a draft,
-so that you never lose your work. Drafts are saved for 30 days.
+Zulip lets you save the content of a message as a draft when you close the
+compose box, ensuring that you never lose your work. Drafts are saved for 30
+days.
 
 !!! warn ""
 
-    **Note**: Drafts are local to your client, and are not synced to
-    other devices and browsers.
+    **Note**: Drafts are stored on your device and are not automatically synced
+    with other devices and browsers.
```

Comment:
Yeah, this is the [CZO conversation](https://chat.zulip.org/#narrow/stream/101-design/topic/drafts.20in.20sidebar/near/1274278) that I was recalling, and is what I based my updates for this page on last summer (#22364).

We could maybe take out the "local to your client" and just mirror what's on the web-app for the drafts overlay. Looking at the screenshots from the PR I linked to, I probably just kept that first part. If we can think of a less technical way of saying that, then great. But otherwise I would keep it.